packages/thirdweb/src/wallets/injected/index.ts

```diff
@@ -272,6 +272,7 @@ export class InjectedWallet implements Wallet {
               accessList: tx.accessList,
               value: tx.value ? toHex(tx.value) : undefined,
               gas: tx.gas ? toHex(tx.gas) : undefined,
+              value: tx.value ? toHex(tx.value) : undefined,
```

Comment:
we're double adding it here, that will break the build (it's already there above on line 273)
```suggestion
```

Reply:
Merge issue. Will remove thanks!